ansible_collections/arista/avd/docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 mkdocs
-mkdocs-material==9.0.15
+mkdocs-material<10
```

Comment:
why the cap? (which is much better than the hardcoded version ;) )
Can we lift it in a next PR?

Reply:
Who knows that they will break in the next version...

Reply:
If you prefer I can remove the cap.

Reply:
let's keep it - I thought you were capping because it was already broken in next version and was curious of the why but actually 10.0 is not out yet